config-ui/src/styles/nav.scss

```
@@ -17,11 +17,43 @@
   position: fixed;
   top: 0;
   border-radius: 0;
-  background: #ffffff;
   box-shadow: none;
   z-index: 1;
 }
 
+.navbarMenuButton{
+  position: absolute;
+  display: none;
+  color: #7497F7;
+  &:hover {
+    color: #E8471C; 
+  }
+
+  @media only screen and (max-width: 850px) {
+    display: flex;
+    left: 0px;
+  }
+}
+
+.navbarMenuButtonSidebarOpened{
```

This css rule is missing a space before the opening bracket `{`
```suggestion
.navbarMenuButtonSidebarOpened {
```